src/components/bodyImage.tsx

```diff
@@ -0,0 +1,76 @@
+// ----- Imports ----- //
+
+import React, { FC, ReactNode } from 'react';
+import { css, SerializedStyles } from '@emotion/core';
+import { text } from '@guardian/src-foundations/palette';
+import { textSans } from '@guardian/src-foundations/typography';
+import { between, from } from '@guardian/src-foundations/mq';
+import { remSpace, breakpoints } from '@guardian/src-foundations';
+
+import Image, { Props as ImageProps } from 'components/image';
+import { textPadding } from 'styles';
+import { Pillar, PillarStyles, getPillarStyles } from 'pillar';
+
+
+// ----- Setup ----- //
+
+const sizes = `(min-width: ${breakpoints.phablet}px) 620px, 100vw`;
+
+
+// ----- Component ----- //
+
+interface Props {
+    image: Omit<ImageProps, 'sizes'>;
+    figcaption: ReactNode;
+    pillar: Pillar;
+}
+
+const styles = css`
+    margin: 1rem 0 ${remSpace[3]};
+
+    ${from.wide} {
+        margin-bottom: 1rem;
+    }
+
+    img {
+        display: block;
+        width: 100%;
+
+        ${between.phablet.and.wide} {
+            margin: 0 ${remSpace[2]};
+        }
+    }
+`;
+
+const triangleStyles = ({ kicker }: PillarStyles): SerializedStyles => css`
+    fill: ${kicker};
+    height: 0.8em;
+    padding-right: ${remSpace[1]};
+`;
+
+const captionStyles = css`
+    ${textSans.xsmall()}
+    ${textPadding}
+    padding-top: ${remSpace[2]};
+    color: ${text.supporting};
+`;
+
+const BodyImage: FC<Props> = ({ figcaption, image, pillar }: Props) =>
+    <figure css={styles}>
+        <Image {...image} sizes={sizes} />
+        <figcaption css={captionStyles}>
+            <svg
```

Comment:
This seems like it could be reusable with dotcom and elsewhere, maybe one for the design system? @SiAdcock

Reply:
Yup, long term it may be able to go [here](https://github.com/guardian/source/tree/master/src/core/svgs).